appeals/web/src/server/appeals/appeal-details/representations/document-attachments/add-document.router.js

```diff
@@ -0,0 +1,49 @@
+import { Router as createRouter } from 'express';
+import { asyncHandler } from '@pins/express';
+import {
+	renderDocumentUpload,
+	postDocumentUpload,
+	renderDateSubmitted,
+	renderRedactionStatus,
+	postDateSubmitted,
+	postRedactionStatus
+} from './add-document.controller.js';
+import { saveBodyToSession } from '#lib/middleware/save-body-to-session.js';
+import {
+	createDateInputDateInPastOrTodayValidator,
+	createDateInputDateValidityValidator,
+	createDateInputFieldsValidator
+} from '#lib/validators/date-input.validator.js';
+import { validateRedactionStatus } from '../representations.validators.js';
+import {
+	postCheckYourAnswers,
+	renderCheckYourAnswers
+} from '../../final-comments/controller/check-your-answers.js';
+
+const router = createRouter({ mergeParams: true });
+
+router.get('/', asyncHandler(renderDocumentUpload));
```

Comment:
very minor, but by convention I think the .post should chain from the .get, eg: `router.get(...).post(...);` rather than `router.get(...); router.post(...);`